miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -468,6 +517,33 @@ var HotUtils = {
     });
   },
   
+  showServerErrors: function (response, serverStatus) {
+    var responseText = JSON.parse(response.responseText);
+    var alreadyShown = HotUtils.serverErrors.filter(function (error) {
+      if (error.status == serverStatus && error.message == responseText.detail) {
+        // if it exists, update the number
+        error.rows++;
+        return true;
+      } else {
+        return false;
+      }
+    });
+    if (!alreadyShown.length) {
+      // add this response to the server errors to display
+      HotUtils.serverErrors.push({ rows: 1, message: responseText.detail, status: serverStatus });
+    }
+    
+    if (HotUtils.serverErrors.length) {
+      var serverErrorMessages = document.getElementById('serverErrors');
+      
+      serverErrorMessages.innerHTML = '<ul>' + HotUtils.serverErrors.map(function(error) {
+        return '<li>' + error.rows + ' row' + (error.rows == 1 ? '' : 's') + ': ' + error.status + '. ' + error.message + '</li>';
```

Comment:
This syntax would result in a message like:
`Bad Request. Must give search terms to look up identities on 2 rows`
`Internal Server Error. NullPointerException on 2 rows`
That doesn't quite parse correctly to me because it's unclear that the "on 2 rows" is independent of the error message.
Perhaps something like the following?
`2 Bad Request errors. Must give search terms to look up identities`
`2 Internal Server Error errors. NullPointerException`